classes/class04/homework/app.js

```diff
@@ -0,0 +1,44 @@
+var express = require('express');
+var path = require('path');
+var logger = require('morgan');
+var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser');
+var exphbs  = require('express-handlebars');
+var PORT = process.env.PORT || 3000;
+
+var ingredients = require('./routes/ingredients')
+var order = require('./routes/order')
+var kitchen = require('./routes/kitchen')
+
+var app = express();
+
+app.engine('handlebars', exphbs({defaultLayout: 'main'},'empty'));
+app.set('view engine', 'handlebars');
+
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: false}));
+app.use(cookieParser());
+app.use(express.static(path.join(__dirname, 'public')));
+
+app.get('/',function(req,res){
+	res.render("home",{'paths':['/ingredients','/order','/kitchen']});
```

Comment:
Yeah, these routes are short, but it's still nice to keep them all in one place, out of app.js.